pkg/i18n/english.go

```diff
@@ -1050,6 +1052,8 @@ func EnglishTranslationSet() TranslationSet {
 		PushTag:                             "Push tag",
 		CreateTag:                           "Create tag",
 		CreatingTag:                         "Creating tag",
+		ForceTag:                            "Force Tag",
+		ForceTagPrompt:                      "The tag '{{.tagName}}' exists already. Press 'esc' to cancel, or 'enter' to overwrite.",
```

Comment:
I think we're better off just saying 'The tag '...' exists already. Overwrite?' Cos the keybindings may be overwritten

Reply:
I copied this from what we show when force-pushing: "Your branch has diverged from the remote branch. Press 'esc' to cancel, or 'enter' to force push."

So it should be changed there, too. Or we substitute in the actual keybindings, like we did [here](https://github.com/jesseduffield/lazygit/pull/2777). I'm very undecided what I like better, let me know what you prefer.

Reply:
I decided to go with the second option; see 7e7ac856a540 and d80165fa4f8c.